cli/drivers/ValetDriver.php

```diff
@@ -177,4 +177,28 @@ protected function isActualFile($path)
     {
         return ! is_dir($path) && file_exists($path);
     }
+
+    /**
+     * Load server environment variables if available.
+     *
+     * @param  string  $sitePath
+     * @param  string  $siteName
+     * @return void
+     */
+    public function loadServerEnvironmentVariables($sitePath, $siteName)
+    {
+        $varFilePath = $sitePath . '/.valet-env.php';
+        if (! file_exists($varFilePath)) {
+            return;
+        }
+
+        $variables = include $varFilePath;
+        if (! isset($variables[$siteName])) {
+            return;
+        }
+
+        foreach ($variables[$siteName] as $key => $value) {
+            $_SERVER[$key] = $value;
```

Comment:
Why not using `putenv($key.'='.$value)` here along with the already existing `$_SERVER`? 🤔

Reply:
I was only trying to mimic Apache's way of loading env vars on the vhosts files, and if I'm not mistaken, that only sets values on`$_SERVER`.
I don't know if duplicating the values on both `$_SERVER` and `$_ENV` is useful or desirable, but as you say, we can add that single line to the loop if we do.

What say the rest of us?